### PR TITLE
Webpack task view code

### PR DESCRIPTION
**Pull request by jrbotros**

### CODE REVIEW

File: orchestra/static/dist/main.css [hunk 1]
```diff
@@ -1334,6 +1334,20 @@ li.checklist-item, .new-item {
 
 .ql-snow a {
```
Comment (marcua) [inline orchestra/static/dist/main.css]: Remind me why this css file is in `dist` again?

File: orchestra/static/dist/main.css [hunk 2]
```diff
@@ -1355,3 +1369,183 @@ li.checklist-item, .new-item {
...
+  font-size: 1.5rem; }
+
+/* Fixed-location informational boxes */
+.fixed-infobox-outer {
+  margin: 20px auto;
+  position: fixed;
+  top: 160px;
+  left: 0;
+  right: 0;
+  z-index: 1009; }
```
Comment (marcua) [inline orchestra/static/dist/main.css]: i know ending on the same line used to be our style, but i'd vote for closing on the next line :)

File: orchestra/templates/orchestra/index.html [hunk 3]
```diff
@@ -36,10 +36,6 @@
 <!-- TODO(jrbotros): remove other scripts once they've been bundled properly -->
 <script src="{% static "dist/main.js" %}" type="text/javascript"></script>
 
-<script src="{% static "orchestra/task/js/controllers.js" %}" type="text/javascript"></script>
```
Comment (marcua) [inline orchestra/templates/orchestra/index.html:39]: <3

**Comment by marcua** [comment]
LGTM

File: orchestra/templates/orchestra/index.html (hunk 3 above)
Comment (jrbotros) [reply orchestra/templates/orchestra/index.html:39]: This now lives in `dist/main.js`

**Pull request merged**